gradle/libs.versions.toml

```diff
@@ -7,20 +7,23 @@ s3-mock = "2.11.0"
 retrofit = "2.9.0"
 gson = "2.10.1"
 protobuf = "3.25.1"
+okhttp-bom = "4.12.0"
 
 [libraries]
 kotlin-bom = { group = "org.jetbrains.kotlin", name = "kotlin-bom", version.ref = "kotlin" }
 kotlin-stdlib = { group = "org.jetbrains.kotlin", name = "kotlin-stdlib-jdk8", version.ref = "kotlin" }
 amazon-bom = { group = "software.amazon.awssdk", name = "bom", version.ref = "amazon-bom" }
-amazon-s3 = { group = "software.amazon.awssdk", name = "s3", version.ref = "amazon-bom" }
-amazon-sso = { group = "software.amazon.awssdk", name = "sso", version.ref = "amazon-bom" }
-amazon-sts = { group = "software.amazon.awssdk", name = "sts", version.ref = "amazon-bom" }
+amazon-s3 = { group = "software.amazon.awssdk", name = "s3" }
```

Comment:
Just removed the direct version refs as these dependencies will be resolved via the `bom` anyway